frontend/src/components/login/login-callback/LoginCallback.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext, useEffect, useState } from 'react'
+
+import { AuthContext } from 'contexts/auth.context'
+
+import { getUser, loginWithSgid, setUserAnalytics } from 'services/auth.service'
+
+import { sendException } from 'services/ga.service'
+
+const DASHBOARD_PAGE = '/'
+const LOGIN_PAGE = '/sgid-login'
```

Comment:
Repeat comment on `/ogp-login` as opposed to `/sgid-login`

Reply:
Same as above